package.json

```diff
@@ -65,6 +65,14 @@
       "description": "Have eslint attempt to fix some errors automatically when saving the file.",
       "type": "boolean",
       "default": false
+    },
+    "scopes": {
+      "title": "Language scopes to use linter",
+      "type": "array",
```

Comment:
Add a description along the lines of [this](https://github.com/AtomLinter/linter-proselint/blob/483fa1d7c65f7f83a1995a7aa4b546a734378bab/package.json#L15) describing what this setting does, and how the user can get the information to update it.